app/utils/query_info.js

```
@@ -0,0 +1,21 @@
+const COPY_FORMATS = ['TEXT', 'CSV', 'BINARY'];
```

If the user specifies an invalid format (eg: `WADUS`), we will set the format to `TEXT`. Shouldn't we return an error, or at least a `false`

---

If the user sends a bad format, the request will fail. 

But for sure, this module should work independently. I am going to change it

---

I have made some changes... not perfect, but enough for me